app/services/cocina/mapper.rb

```diff
@@ -131,9 +131,17 @@ def build_identification
       end
     end
 
+    # rubocop:disable Rails/DynamicFindBy
     def build_descriptive
-      { title: [{ status: 'primary', value: TitleMapper.build(item) }] }
+      desc = { title: [{ status: 'primary', value: TitleMapper.build(item) }] }
+
+      # collections are registered with abstracts
+      return desc if item.descMetadata.find_by_terms(:abstract).blank?
```

Comment:
I find this easier to read if we avoid the low-level api (e.g. find_by_terms):

```suggestion
      return desc if item.descMetadata.abstract.blank?
```